src/main/kotlin/org/objectionary/deog/steps/CondAttributesSetter.kt

```diff
@@ -42,47 +41,47 @@ internal class CondAttributesSetter(
     private fun processApplications() {
         conditions.forEach { node ->
             var tmpNode = node.firstChild.nextSibling
-            var line = line(tmpNode)
+            var line = tmpNode.getAttrContent("line")
             val cond: MutableList<Node> = mutableListOf(tmpNode)
-            while (line(tmpNode.nextSibling.nextSibling) == line) {
+            while (tmpNode.nextSibling.nextSibling.getAttrContent("line") == line) {
                 cond.add(tmpNode.nextSibling.nextSibling)
                 tmpNode = tmpNode.nextSibling.nextSibling
-                line = line(tmpNode)
+                line = tmpNode.getAttrContent("line")
             }
             tmpNode = tmpNode.nextSibling.nextSibling
             val fstOption: MutableList<Node> = mutableListOf(tmpNode)
-            while (line(tmpNode.nextSibling.nextSibling) == line) {
+            while (tmpNode.nextSibling.nextSibling.getAttrContent("line") == line) {
                 fstOption.add(tmpNode.nextSibling.nextSibling)
                 tmpNode = tmpNode.nextSibling.nextSibling
-                line = line(tmpNode)
+                line = tmpNode.getAttrContent("line")
             }
             tmpNode = tmpNode.nextSibling.nextSibling
             val sndOption: MutableList<Node> = mutableListOf(tmpNode)
-            while (line(tmpNode.nextSibling.nextSibling) == line) {
+            while (tmpNode.nextSibling.nextSibling.getAttrContent("line") == line) {
                 sndOption.add(tmpNode.nextSibling.nextSibling)
                 tmpNode = tmpNode.nextSibling.nextSibling
-                line = line(tmpNode)
+                line = tmpNode.getAttrContent("line")
             }
-            val dgCond = DgNodeCondition(cond)
-            traverseParents(node.parentNode, dgCond.freeVars)
-            name(node)?.let { name ->
+            val igCond = DgNodeCondition(cond)
```

Comment:
@c71n93 Can you please make this variable name more readable?

Reply:
@Graur done